### ISSUE TITLE
Using playlist URL -- RegexNotFoundError: Unable to extract title

### ISSUE THREAD

**Issue by kodekrash** [rg3/youtube-dl#7545]
When attempting to download videos using a playlist, I get the following error. Attempted with several different playlists with same issue. I can download random individual tracks from the playlist without issue.

```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-f', u'bestaudio', u'-x', u'--audio-format', u'wav', u'--audio-quality', u'0', u'-k', u'-o', u'%(autonumber)s-.%(ext)s', u'--verbose', u'https://www.youtube.com/playlist?list=PL4A77333EA48F41D1']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.11.15
[debug] Python version 2.7.10 - Linux-4.2.5-201.fc22.x86_64-x86_64-with-fedora-22-Twenty_Two
[debug] exe versions: ffmpeg 2.6.4, ffprobe 2.6.4
[debug] Proxy map: {}
[youtube:playlist] PL4A77333EA48F41D1: Downloading webpage
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 661, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 290, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 1644, in _real_extract
    return self._extract_playlist(playlist_id)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 1619, in _extract_playlist
    page, 'title')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 593, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 584, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by jaimeMF** [comment]
Thanks for the report, it will work in the next version.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl version 2015.11.18. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by kodekrash** [comment]
I can confirm this issue does not exist in 2015.11.18.
